BlackJack/index.js

```diff
@@ -29,189 +41,295 @@ makeCard();
   location.reload();
-}else if(firstCardNum + secondCardNum === 'A1' || firstCardNum + secondCardNum === 'AJ' || firstCardNum + secondCardNum === 'AQ' || firstCardNum + secondCardNum === 'AQ' ){
-  alert('BlackJack You Win')
+} else if (
+  firstCardNum + secondCardNum === "A1" ||
+  firstCardNum + secondCardNum === "AJ" ||
+  firstCardNum + secondCardNum === "AQ" ||
+  firstCardNum + secondCardNum === "AQ"
+) {
+  alert("BlackJack You Win");
   location.reload();
 }
 
-function firstCardNumFunc (){
-  let firstCardNum = firstCardN.charAt(0)
-  if(firstCardNum === '2' || firstCardNum === '3' || firstCardNum === '4' || firstCardNum === '5' || firstCardNum === '6' || firstCardNum === '7' || firstCardNum === '8' || firstCardNum === '9'){
-     return firstCardN = firstCardNum;
-  }else if( firstCardNum === 'J' || firstCardNum === 'K' || firstCardNum === 'Q' || firstCardNum === '1'){
-    return firstCardN = '10'
-  }else if(firstCardNum === 'A'){
-  return firstCardN = '1'
+function firstCardNumFunc() {
+  let firstCardNum = firstCardN.charAt(0);
+  if (
+    firstCardNum === "2" ||
+    firstCardNum === "3" ||
+    firstCardNum === "4" ||
+    firstCardNum === "5" ||
+    firstCardNum === "6" ||
+    firstCardNum === "7" ||
+    firstCardNum === "8" ||
+    firstCardNum === "9"
+  ) {
+    return (firstCardN = firstCardNum);
+  } else if (
+    firstCardNum === "J" ||
+    firstCardNum === "K" ||
+    firstCardNum === "Q" ||
+    firstCardNum === "1"
+  ) {
+    return (firstCardN = "10");
+  } else if (firstCardNum === "A") {
+    return (firstCardN = "1");
   }
 }
-firstCardNumFunc ()
-console.log(firstCardN)
+firstCardNumFunc();
+console.log(firstCardN);
 
 //save player second card value
-function secondCardNumFunc (){
-  let secondCardNum = secondCardN.charAt(0)
-  if(secondCardNum === '2' || secondCardNum === '3' || secondCardNum === '4' || secondCardNum === '5' || secondCardNum === '6' || secondCardNum === '7' || secondCardNum === '8' || secondCardNum === '9'){
-     return secondCardN = secondCardNum;
-  }else if(secondCardNum === 'J' || secondCardNum === 'K' || secondCardNum === 'Q' || secondCardNum === '1'){
-    return secondCardN = '10'
-  }else if(secondCardNum === 'A'){
-    return secondCardN = '1'
-    }
+function secondCardNumFunc() {
+  let secondCardNum = secondCardN.charAt(0);
+  if (
+    secondCardNum === "2" ||
+    secondCardNum === "3" ||
+    secondCardNum === "4" ||
+    secondCardNum === "5" ||
+    secondCardNum === "6" ||
+    secondCardNum === "7" ||
+    secondCardNum === "8" ||
+    secondCardNum === "9"
+  ) {
+    return (secondCardN = secondCardNum);
+  } else if (
+    secondCardNum === "J" ||
+    secondCardNum === "K" ||
+    secondCardNum === "Q" ||
+    secondCardNum === "1"
+  ) {
+    return (secondCardN = "10");
+  } else if (secondCardNum === "A") {
+    return (secondCardN = "1");
+  }
 }
-secondCardNumFunc ()
-console.log(secondCardN)
+secondCardNumFunc();
+console.log(secondCardN);
 
```

Comment:
This is needed because you assigned a string value. You can skip this step by assigning a integer directly to it.